tests/unit/aiplatform/test_metadata_schema_types.py

```diff
@@ -0,0 +1,576 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think test_experiments_copy (seprate file) is unintended

Reply:
Done. good catch :)